### PR TITLE
Support file method simulations

### LINKED ISSUE #505: Enable calling auth-protected methods for testing
### Goal

Set of functions that enables us to call auth-protected methods

### Context

Some collateral auctions simulations (such as `UNIV2DAIUSDC-A`, `WSTETH-B`) can't be currently performed due to some settings in the vault/liquidation-relevant contracts. To enable it we have to either directly edit those changes, enable us to call `file`-like administration methods or in other way provide us with ability to change them (eg compile, vote, execute the spell, like it's done the proper way).

### Assets

- Overview issue #501

### Tasks

- [x] Research who has administrator privileges, how to get them, report it here
- [x] Propose the implementation & discuss
- [x] Implement

### PR DESCRIPTION
**Pull request by KirillDogadin-std**
Closes #505

Checklist:
- [x] issue number linked above after pound (`#`)
    - replace "Closes " with "Contributes to" or other if this PR does not close the issue
- [x] issue checkboxes are all addressed
- [x] manually checked my feature / not applicable
- [x] wrote tests / not applicable
- [x] attached screenshots / not applicable

### CODE REVIEW

**Review by LukSteib — APPROVED** [review]
With reference to the collateral overview here (https://github.com/sidestream-tech/unified-auctions-ui/issues/501#issuecomment-1282125330) I've tested the following simulations: 

- WSTETH-B ✔️ 
- UNIV2DAIUSDC-A ✔️

File: core/simulations/helpers/createVaultWithCollateral.ts [hunk 1]
```diff
@@ -21,11 +22,16 @@ import { CollateralConfig, CollateralType } from '../../src/types';
...
+    setCollateralDebtCeilingToGlobal,
+    setCollateralLiquidationLimitToGlobal,
+} from '../../helpers/hardhat/contractParametrization';
+import { overwriteStabilityFeeAccumulationRate } from '../../helpers/hardhat/overwrites';
 
 const UNSUPPORTED_COLLATERAL_TYPES = [
     'CRVV1ETHSTETH-A', // Collateral handled differently
-    'UNIV2DAIUSDC-A', // Liquidation limit too high (fails with "Dog/liquidation-limit-hit")
-    'WSTETH-B', // Does not accumulate stability fee rate at all
     'RETH-A', // [temporary] this collateral is not yet deployed, tested via different flow
```
Comment (valiafetisov) [inline core/simulations/helpers/createVaultWithCollateral.ts:35]: The suggestion during some of our dailies was to remove `RETH-A` from here since we now (after it was deployed) should be able to test it as well using `file` methods. But we can also do it in another PR!

**Review by valiafetisov — APPROVED** [review]
